src/compositions/useLspFactory.ts

```diff
@@ -0,0 +1,12 @@
+import { getSigner } from "@/services/provider.service";
+import { LSPFactory } from "@lukso/lsp-factory.js";
+
+export async function useLspFactory(): Promise<LSPFactory> {
+  const { provider } = await getSigner();
+  const { chainId } = await provider.getNetwork();
+  return new LSPFactory(provider as unknown as string, {
```

Comment:
Let's grab `provider` from config and `chainId` from store. Please make sure to do some check for those variables and throw error otherwise (+ tests for this ;)